test/tests/tests.cpp

```diff
@@ -0,0 +1,13 @@
+// This serves as the entry point when building for MSVC which compiles gtest
+// directly into the test binary.
+#ifdef _MSC_VER
+
+#include <gtest/gtest.h>
+
+int main(int argc, char * * argv)
+{ 
+    testing::InitGoogleTest(&argc, argv); 
+    RUN_ALL_TESTS(); 
```

Comment:
why this and not https://github.com/google/googletest/blob/master/googletest/src/gtest_main.cc?

If you want to keep this, make it return value.